File: Leetcode/res/Move Zeroes/6.cpp
\*
Author: allannozomu
Runtime: 20 ms
Memory: N/A*\

class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int slow = 0;
        for (int fast = 0; fast < nums.size(); ++fast){
            if (nums[fast] != 0){
                nums[slow] = nums[fast];
                slow++;
            }
        }
        for (; slow < nums.size(); ++slow){
            nums[slow] = 0;
        }
    }
};

File: Leetcode/res/Subsets/6.py
# Author: allannozomu
# Runtime: 44 ms
# Memory: 13.2 MB

class Solution:
    def subsets(self, nums: List[int]) -> List[List[int]]:
        resp = []
        for num in nums:
            new_resp = 
            for r in resp:
                new_resp.append(r)
                new_resp.append(r + [num])
            resp = new_resp
        return resp